### ISSUE TITLE
Curie hráči tyká i vyká

### ISSUE THREAD

**Issue by rado-p** [prekladyher/fallout4-cestina#354]
![fallout 4 screenshot 2018 01 12 - 17 41 20 07](https://user-images.githubusercontent.com/22203490/34885889-a2e2907e-f7c1-11e7-9918-2b90b5e924d1.png)

**Comment by pavelhoral** [comment]
Tato issue vznikla na základě poznámky pro překlad ze slovníčku:

> Nebál bych se u ní dát jako základ vykání - i když se z ní stane člověk, mluví naprosto identicky (dialogy pro člověka a stroj jsou stejné). Je slušná, uctivá... asi se to nebude hodit všude, ale když tu mám dialogy, které zahujeje frázemi "Pardom me, monsieur" - tak tam fakt tykání dávat nebudu. Více se mi k ní hodí to naše základní pravidlo pro roboty.

Tohle je ale nesmysl, protože půlka dialogů Curie je sdílených mezi její robotí a synetickou verzí. Tuto issue tedy spíše berme jako požadavek na zrušení výše uvedené poznámky.

**Comment by Dumec** [comment]
No, podle mě je nesmysl, aby Curie tykala – v robotické i syntetické verzi (dialogy jsou stejné, což tam taky píšu). A teď vůbec neřeším nějaký děravé plošné pravidlo o robotech, řeším styl mluvy a osobnost Curie. Je mi šumák, co máme napsané pro "roboty" – tak, jak čtu a slyším její vyjadřování v angličtině, nemůžu to přeložit jako tykání... o tomle v té poznámce šlo :)

**Comment by MANUS-manww** [comment]
Tady by teda mělo být vykání... Pokud je slovníček aktuální....

**Comment by rado-p** [comment]
Teď je to namíchané. Někde tyká někde vyká. Problém je že později z ní bude syntetik a netuším jak to je s "románkem" Jak psal Palko: Spát s někým kdo nám vyká....

**Comment by MANUS-manww** [comment]
Pak bych dal vykání pro robota a tykání pro syntha....(je to přeci podmíněné nějakým questem - aby se z ní stal synth...)

**Comment by pavelhoral** [comment]
@MANUS-manww Ty stále nechápeš, že ty dialogy jsou sdílené. Jeden dialog, jeden překlad, jedny titulky, dvě verze Curie, dva hlasy.

**Comment by MANUS-manww** [comment]
Aha takže buď tykání nebo vykání...

**Comment by StoupaJ** [comment]
No tohle je už nějakej čas, co jsem projížděl komplet Curie dialogy, ale:
Kdysi jsme se o tom poměrně důkladně bavili a nakonec se dohodlo, že ještě v rámci prvního kontaktu hráče s Curie bude vykat - od momentu, kdy ve Vaultu padne nabídka Curie jako společnice, pozvolna přejde na tykání a to už tak zůstane a v podstatě jediné její vykací věty budou s francouzskými osloveními. --> Dalo to poměrně dost práce tohle vychytat :( Navíc Curie má skoro 1700 replik - tohle předělávat zase jinak by bylo na palici... (ale i z toho důvodu je samozřejmě možné, že nějaká věta "ulítla"...)
